src/scitokens_internal.cpp

```diff
@@ -835,6 +925,11 @@ Validator::get_public_key_pem(const std::string &issuer, const std::string &kid,
     result->m_issuer = issuer;
     result->m_kid = kid;
 
+    // Track this issuer for background refresh if enabled
+    if (configurer::Configuration::get_background_refresh_enabled()) {
+        internal::BackgroundRefreshManager::get_instance().add_issuer(issuer);
```

Comment:
This will be removed on revision - the issuers will just be taken from the database.